register/blake2/multihash_blake2.go

```diff
@@ -0,0 +1,61 @@
+/*
+	This package has no purpose except to perform registration of multihashes.
+
+	It is meant to be used as a side-effecting import, e.g.
+
+		import (
+			_ "github.com/multiformats/go-multihash/register/blake2"
+		)
+
+	This package registers several multihashes for the blake2 family
+	(both the 's' and the 'b' variants, and in a variety of sizes).
+*/
+package blake2
+
+import (
+	"hash"
+
+	"github.com/minio/blake2b-simd"
+	"golang.org/x/crypto/blake2s"
+
+	"github.com/multiformats/go-multihash"
+)
+
+const (
+	BLAKE2B_MIN = 0xb201
+	BLAKE2B_MAX = 0xb240
+	BLAKE2S_MIN = 0xb241
+	BLAKE2S_MAX = 0xb260
```

Comment:
These should all go in a shared package somewhere.

Reply:
I dunno how I feel about these.  I kind of just cargo-culted them over in the first pass.  Looking again now... they seem like an implementation detail that might not be sensible to export at all?  I can't imagine anyone but us looping over these ranges.  (And we don't even use the 2B ranges, at that.)

Reply:
Yeah, I guess... We can always fix this later if necessary.

Maybe don't export them then?

Reply:
Done.